packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -82,12 +83,13 @@ export const useRadio = (props: RadioProps): RadioState => {
     props: {
       value,
       label,
+      subtext,
       ...rest,
       ref: buttonRef,
       ...pressable.props,
       accessible: true,
       accessibilityRole: 'radio',
-      accessibilityLabel: accessibilityLabel ?? label,
+      accessibilityLabel: accessibilityLabel ?? (subtext ? label + subtext : label),
```

Comment:
Do you maybe need to add a space or something here between label and subtext

Reply:
Thanks for catching this! Consulted @ksiler and we decided to put subtext in accessibilityHint.